### ISSUE TITLE
ldap ext with external oidc provider

### ISSUE THREAD

**Issue by cheoalfredo** [Nordes/IdentityServer4.LdapExtension#26]
hi, i´m trying to setup Azure AD as external provider in a deployment i have with LdapExtension, but the callback allways ends up calling the Search method and obviusly the user(Azure AD) does not exist in any ldap and the login fail, can we set up the ldap extension and also external oidc ??

**Comment by nobba75** [comment]
I second this, the LdapExtension interferes with other external provider logins. Any workaround? Solution?

**Comment by Nordes** [comment]
No, sorry. I don't have any workaround. I haven't played much with this project for the last year. It was at its beginning a PoC to prove that it was not so hard to add any LDAP to Identity Server. Since it was not documented anywhere, I did it so that it can help others.

However, if you wish to fix this issue, please do a PR.
